swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -17,6 +17,8 @@ enum PacketTunnelProviderError: Error {
 
 class PacketTunnelProvider: NEPacketTunnelProvider {
   private var adapter: Adapter?
+  private var hasReceivedFirstConfig = false
+  private var networkSettings: NetworkSettings?
```

Comment:
It seems the `networkSettings` have moved to this file. Is there a reason why it couldn't be in `Adapter` anymore?

Reply:
moved it back